### PR TITLE
Migrate to peewee moves

### PR DESCRIPTION
**Pull request by AMecea**
- [x] needs documentation update

### CODE REVIEW

File: colibris/conf/settings.py [hunk 1]
```diff
@@ -52,8 +52,8 @@
         'handlers': ['console']
     },
     'loggers': {
-        'peewee_migrate': {
-            'level': 'DEBUG'  # Always show details when doing migrations
+        'peewee_moves': {
+            'level': 'DEBUG'   # Always show details when doing migrations
```
Comment (ccrisan) [inline colibris/conf/settings.py:56]: Why more than 2 spaces?

**Comment by ccrisan** [comment]
What's the purpose of the added `__packagename__.db` file?

**Comment by AMecea** [comment]
Thanks, @ccrisan for the review. I've updated the code accordingly. Also, I've added documentation of how to use Peewee Moves and how to upgrade from Colibris v0.9 to v0.10.

**Comment by AMecea** [comment]
Hi @ccrisan, Now, peewee-moves have fake migrations support. This PR is ready for review now. Thanks!

File: docs/source/Migrations.md [hunk 2]
```diff
@@ -1,17 +1,137 @@
 # Migrations
 
+
+## Manage Migrations
+The migrations are managed by [`peewee-moves`](https://github.com/timster/peewee-moves) a small and
```
Comment (ccrisan) [inline docs/source/Migrations.md:5]: I recommend a `,` comma before `a small...`

File: docs/source/Migrations.md [hunk 3]
```diff
@@ -1,17 +1,137 @@
...
 
 To apply migrations on the currently configured database, use:
 
-    ./${PACKAGE}/manage.py migrate
+    ./${PACKAGE}/manage.py db upgrade
+    
+
+## Upgrade model
+
+Peewee Moves support only creating a new model. The alteration of an existing model is done manually
```
Comment (ccrisan) [inline docs/source/Migrations.md:46]: `supports`

File: docs/source/Migrations.md [hunk 4]
```diff
@@ -1,17 +1,137 @@
...
+by generating a new revision and editing it. For generating an empty migration use:
+
+    ./${PACKAGE}/manage.py db revisions alter_model
+    
+Then update the newly generated file `migrations/0XXX_alter_model.py` to contains the model changes.
```
Comment (ccrisan) [inline docs/source/Migrations.md:51]: `to contain`

File: docs/source/Migrations.md [hunk 5]
```diff
@@ -1,17 +1,137 @@
...
 To apply migrations on the currently configured database, use:
 
-    ./${PACKAGE}/manage.py migrate
+    ./${PACKAGE}/manage.py db upgrade
+    
+
+## Upgrade model
+
+Peewee Moves support only creating a new model. The alteration of an existing model is done manually
+by generating a new revision and editing it. For generating an empty migration use:
```
Comment (ccrisan) [inline docs/source/Migrations.md:47]: `, use`

File: docs/source/Migrations.md [hunk 6]
```diff
@@ -1,17 +1,137 @@
...
+    
+Then update the newly generated file `migrations/0XXX_alter_model.py` to contains the model changes.
+The documentation of `peewee-moves` Migrator API can be found
+[here](https://peewee-moves.readthedocs.io/en/latest/usage.html#migrator-api).
+
+
+
+### Adding/Removing/Renaming a model field
+
+Create an empty revision file with the above command `migrations/0004_alter_user_fields.py` then
```
Comment (ccrisan) [inline docs/source/Migrations.md:59]: Please add some commas to make this sentence more intelligible.

File: docs/source/Migrations.md [hunk 7]
```diff
@@ -1,17 +1,137 @@
...
+    migrator.add_column('user', 'display_name', 'char', max_length=58, null=True)
+    
+    # Remove role field
+    migrator.drop_column('user', 'role')
+    
+    # Rename key field to password
+    migrator.rename_column('user', 'key', 'password')
+
+def downgrade(migrator):
+    # Remove dispaly name field
```
Comment (ccrisan) [inline docs/source/Migrations.md:74]: `display_name`

File: docs/source/Migrations.md [hunk 8]
```diff
@@ -1,17 +1,137 @@
...
+    migrator.drop_column('user', 'display_name')
+    
+    # Add role field
+    migrator.add_column('user', 'role', 'char', max_length=10)
+    
+    # Rename
```
Comment (ccrisan) [inline docs/source/Migrations.md:80]: `Rename password field to key.

File: docs/source/Migrations.md [hunk 9]
```diff
@@ -1,17 +1,137 @@
...
+    migrator.execute_sql('CREATE INDEX idx_user_email ON user (email)')
+
+def downgrade(migrator):
+    migrator.execute_sql('DROP INDEX idx_user_email')
+```
+
+
+### Python script
+
+Peewee Migrate allows running SQL queries directly in the migrations. Here is an example of how to
```
Comment (ccrisan) [inline docs/source/Migrations.md:101]: Isn't this a copy-paste leftover?

File: docs/source/Migrations.md [hunk 10]
```diff
@@ -1,17 +1,137 @@
...
+
+
+def downgrade(migrator):
+    migrator.drop_column('user', 'display_name')
+```
+
+
+## Upgrade from peewee-migrate
+
+Colibris version `<=0.9` used for migrations `peewee-migrate` but starting with version `0.10` it
```
Comment (ccrisan) [inline docs/source/Migrations.md:121]: I would add some commas and reorganize sentence topic, e.g.:

    Colibris version `<=0.9` used `peewee-migrate` for migrations, but starting with version `0.10`, it uses `peewee-moves`, a more lightweight manager.

**Comment by ccrisan** [comment]
@AMecea aside from a few grammar mistakes, LGTM.

**Pull request merged**